README.md

```diff
@@ -30,6 +30,29 @@ For example, the following bulids RebbleOS on Debian Stretch:
     cd FreeRTOS-Pebble
     make
 
+The Pebble SDK is a prerequisite for portions of RebbleOS. As an
+example, on Debian, one may create an installation directory for
+hte SDK using the following.
+
+    mkdir ~/pebble-dev/
+    cd ~/pebble-dev/
+    wget https://s3.amazonaws.com/assets.getpebble.com/pebble-tool/pebble-sdk-4.5-linux64.tar.bz2
+    tar -jxf pebble-sdk-4.5-linux64.tar.bz2
+    echo 'export PATH=~/pebble-dev/pebble-sdk-4.5-linux64/bin:$PATH' >> ~/.bash_profile
+    . ~/.bash_profile
+    sudo apt-get install python-pip python2.7-dev python-gevent
+    sudo pip install virtualenv
+    cd ~/pebble-dev/pebble-sdk-4.5-linux64
+    virtualenv --no-site-packages .env
+    source .env/bin/activate
+    pip install -r requirements.txt
+    deactivate
```

Comment:
Adding a hyperlink to the Pebble SDK install page (so OS X users can vary it appropriately, etcetera) would be good, too.